website/docs/en-US/switch.md

```diff
@@ -143,7 +143,7 @@ Switch is used for switching between two opposing states.
 
 | Attribute      | Description          | Type      | Accepted Values       | Default |
 |-----| ----| ----| ----|---- |
-| value / v-model | binding value | boolean / string / number | — | — |
+| value / v-model | binding value,that must be equal to active-value or inactive-value and default to a Boolean type | boolean / string / number | — | — |
```

Comment:
Use `should` to make it more soft.

it should be equivalent to either `active-value` or `inactive-value`, by default it's `boolean` type